app/code/Magento/Customer/i18n/en_US.csv

```diff
@@ -1,23 +1,23 @@
-"Sign Out","Sign Out"
```

Comment:
@swnsma 
To discard these files I have run the following command and do git push, Is this the valid thing that I have done? I am worried because of in second last commit i had only one test fall and now its become to eight.

git rm --cached app/etc/NonComposerComponentRegistration.php
Please help